lib/led/rgb.js

```diff
@@ -193,6 +194,14 @@ function RGB(opts) {
     blue: state.blue
   };
 
+  // state.prev records the last color set using color(),
+  // and is used to determine the new color when calling on() or pulse().
+  state.prev = {
+    red: 255,
+    green: 255,
+    blue: 255
+  };
```

Comment:
Before, `state.prev` was set during calls to `off()` so that you could later call `on()` and restore the LED to its previous color. It was cleared when it was consumed by `on()` and a default value of white (255,255,255) was encoded in that method.

Since nothing else depended on `state.prev`, I've added it to the initial component state with its default white value.

I now update it when `color()` is called, to handle the following case:
```
rgb.color('red');
rgb.pulse(300);
```

I'm also not clearing it when `on()` is called - there's no particular reason to.